deploy/kubernetes/console/templates/NOTES.txt

```diff
@@ -0,0 +1,31 @@
+{{ if .Values.console.techPreview -}}
+Tech Preview is enabled, extra features will be shown.
+
+{{ end }}
+
+{{- $ingress := .Values.console.ingress | default dict -}}
```

Comment:
Could do with some text before just showing the URL - e.g.

"To access Stratos:"